setup.py

```diff
@@ -93,6 +93,22 @@ def checkProgramIsInstalled(self, program, args, where_to_download,
             sys.stderr.write("Error: {}".format(e))
 
 
+install_requires_py = ["numpy >= 1.12.1",
+                       "scipy >= 0.19.0",
+                       "matplotlib >= 2.0.0",
+                       "pysam >= 0.11.2.2",
+                       "intervaltree >= 2.1.0",
+                       "biopython >= 1.68",
+                       "tables >= 3.3.0",
+                       "pandas >= 0.20.2",
+                       "pyBigWig >=0.3.4",
+                       "six >= 1.10.0",
+                       "future >= 0.16.0"
+                       ]
+
+if sys.version_info[0] == 2 or (sys.version_info[0] == 3 and sys.version_info[1] == 4):
+    install_requires_py.append("configparser >= 3.5.0")
```

Comment:
`ConfigParser` was by default in python 2 but it was renamed to `configparser` in python 3. The package `configparser` for python 2 is a backport of the python 3.5 version. To add more confusion: The `configparser` version from 3.4 can be replaced by the backport of the 3.5 version, but then you need to use `from backports import configparser`.

I do not understand these language design decisions. 

https://pypi.python.org/pypi/configparser